### DISCUSSION TITLE
Certification Error when syncing Garmin data

### DISCUSSION THREAD

**Discussion by flyPacific** [yihong0618/running_page#575] [General]
[error.txt](https://github.com/yihong0618/running_page/files/13697441/error.txt)
Hi,
    I like your project and trying to run it locally using my Garmin run data. I hit the cert error during run 
`python3(python) run_page/garmin_sync.py xxxxxxxxxxxxxx(secret_string) --only-run`
    This might not be something with your code and could be my local setting issue. I did some quick search online for this but didn't find much useful info. Wondering did you see the error before running the command? Any suggestion to solve the issue? Thanks.

**Comment by yihong0618** [comment]
@flyPacific seems you are behind `GFW`, this need to run pass `GFW`

**Reply by flyPacific** [reply]
interesting, I am actually located in US so not sure why this can be related to GFW. Did you see any GFW related info from the error?
I am curious did you see similar error before? If not it's more likely related to my local machine setup, in which case I can try to debug myself.
Thanks.

**Reply by yihong0618** [reply]
yes, the location things related GFW.
and you can follow this command to check

```
rm imported.json
rm run_page/data.db
rm GPX_OUT/*
```

then rerun it

**Reply by flyPacific** [reply]
I just tried removing these files (didn't see imported.json though) and reran, it's still the same error.
Any suggestion to run pass GFW in this case (though I still don't understand why it's related to GFW since I am in US right now.)

**Reply by yihong0618** [reply]
SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED]
seems you need to fix this error, please google or ChatGPT it
